<?xml version="1.0"?>
<launch>
  <include file="$(find realsense2_camera)/launch/rs_camera.launch" />

  <arg name="use_rviz" default="false" />
  <group if="$(arg use_rviz)">
      <node name="rviz" pkg="rviz" type="rviz"  args="-d $(find work_vision)/rviz/yolo.rviz"/>
  </group>

  <rosparam file="$(find work_vision)/config/yolov8_params.yaml" command="load" />
  <node name="object_recognition_node"
        pkg="work_vision"
        type="yolov8_node.py"
        output="screen"
        required="true" />

</launch>